src/armor_layers.cpp

```diff
@@ -146,8 +146,8 @@ std::string body_part_names( const std::vector<body_part> &parts )
     std::vector<std::string> names;
     names.reserve( parts.size() );
     for( size_t i = 0; i < parts.size(); ++i ) {
-        const bodypart_id &part = convert_bp( parts[i] ).id();
-        if( i + 1 < parts.size() && parts[i + 1] == static_cast<body_part>( bp_aiOther[part->token] ) ) {
+        const bodypart_id &part = parts[i];
+        if( i + 1 < parts.size() && parts[i + 1] == static_cast<bodypart_id>( bp_aiOther[part->token] ) ) {
```

Comment:
Is it safe to just `static_cast` a `body_part` to a `bodypart_id`?  Judging by your other code, I think you want to use `convert_bp` here.

Reply:
Yes, I missed that one.